src/backend/columnar/columnar_customscan.c

```diff
@@ -62,6 +62,10 @@ typedef struct ColumnarScanState
 typedef bool (*PathPredicate)(Path *path);
 
 
+#define COLUMNAR_CANNOT_PUSHDOWN_EXPR_HEADER_FMT \
+	"columnar planner cannot push down clause \"%s\": "
```

Comment:
I am not completely sure if this makes the code more readable.
It was already embedded 3 times separately before.

By refactoring this shared string out it makes it harder to read the actual message we are emitting, both for reviewing and in the future when we get an error like this to match the error line to a line we see in logs.

Especially with a string interpolation sequence embedded makes it hard from its usage to understand where the formatting goes.

Would you mind to undo this abstraction?